include/smack/BoogieAst.h

```diff
@@ -113,6 +114,17 @@ class BvLit : public Expr {
   void print(std::ostream& os) const;
 };
 
+class FpLit : public Expr {
+	bool neg;
```

Comment:
You are using tabs for indentation here. Change that to two spaces instead. You should never use tabs.